packages/langfuse-wrapper/scripts/setup_langfuse.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# setup_langfuse.sh
+#
+# This script sets up and runs Langfuse locally for development and testing purposes.
+#
+# Key functionalities:
+# 1. Downloads the latest docker-compose.yaml from the Langfuse repository
+# 2. Starts Langfuse using Docker Compose with default initialization variables
+# 3. Waits for the service to be available
+# 4. Launches a browser to open the local Langfuse UI
+# 5. Prints login credentials from the environment file
+#
+# Usage:
+#   ./setup_langfuse.sh
+#
+# Requirements:
+#   - Docker and Docker Compose
+#   - curl
+#   - A .env.langfuse.local file in the env directory
+#
+# Note: This script is intended for local development use only.
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
:nit: `SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")` is a little more straightforward since it avoids invoking an extra subshell. 

if we want to stick with `cd`, could we use `pushd`/`popd` which makes pathing history/traversal easier in the event we need to debug and can get a full pathing history via `dirs`?